### ISSUE TITLE
PAUSE key is not working

### ISSUE THREAD

**Issue by camalouu** [ililim/dual-key-remap#68]
config:
remap_key=F7
when_alone=PAUSE
with_other=PAUSE

pressing F7 does not do anything. Maybe I should use something like MEDIA_PAUSE_PLAY?

**Comment by ililim** [comment]
Yes MEDIA_PAUSE_PLAY would be the right key. Working on the next release, will include all the various media keys as options.

**Comment by ililim** [comment]
Should be fixed in v0.8! https://github.com/ililim/dual-key-remap/releases
